### PR TITLE
Task about me

### PR DESCRIPTION
**Pull request by gidspektor**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,8 +28,16 @@ <h2>Trainee Full Stack Developer</h2>
                  <button type="button"><a href="#project"></a>See my work</button>
            </div>
     </section>
-    <div id="about" class="pages">
-
+    <div id="about" class="pageTwo">
```
Comment (charliecog) [inline index.html]: probably could use a more semantic html tag like a section for this

File: style.css [hunk 2]
```diff
@@ -72,15 +75,12 @@ nav li {
 }
 
 .pageOne {
-    height: 700px;
+    height: 768px;
```
Comment (charliecog) [inline style.css:79]: strangely specific, why 768? lots of blank space at bottom of page?

File: style.css [hunk 3]
```diff
@@ -120,6 +120,28 @@ nav li {
     border: #ffffff;
 }
 
+.me {
+    width: 80%;
+    margin: 0 auto;
+    padding-top: 60px;
```
Comment (charliecog) [inline style.css:127]: this is to stop the white band appearing im guessing, rather than doing this just set the margin top of the h1 to 0

**Review by charliecog — COMMENTED** [review]
few changes to make

File: style.css (hunk 3 above)
Comment (gidspektor) [reply style.css:127]: just to move the whole dive down from the top

**Pull request merged**